### PR TITLE
Recipe for spylon.

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Utility for making PySpark developers happier

### CODE REVIEW

File: recipes/spylon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hashtype }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (jakirkham) [inline recipes/spylon/meta.yaml]: Please add ` --single-version-externally-managed --record=record.txt`.

Comment (mariusvniekerk) [inline recipes/spylon/meta.yaml]: Done

File: recipes/spylon/meta.yaml [hunk 2]
```diff
@@ -25,6 +25,7 @@ requirements:
     - html5lib
     - lxml
     - beautifulsoup4
+    - ca-certificates
```
Comment (jakirkham) [inline recipes/spylon/meta.yaml]: Would love to know the history behind this addition. Maybe there is something flaky on our end that we need to be aware of.

**Comment by jakirkham** [comment]
Thanks @mariusvniekerk.

File: recipes/spylon/meta.yaml (hunk 2 above)
Comment (mariusvniekerk) [inline recipes/spylon/meta.yaml]: Yeah originally as part of my setup.py I fetched some things from https://spark.apache.org and that had ssl cert failures when fetching via urllib.
I altered the package to instead distribute them myself.

**Pull request merged**